### DISCUSSION TITLE
Text Rendering Issues

### DISCUSSION THREAD

**Discussion by Coder2195** [pixijs/pixijs#8128] [Q&A]
So Im trying to make a game, but heres what happened with rendering text
It seems like the viewport messes with pixijs

https://user-images.githubusercontent.com/63069499/150541505-058c04a4-31a9-4ee4-829e-7b51b0e014da.mp4

The code: https://github.com/Coder2195Text/tile-craft

**Comment by bigtimebuddy** [comment]
I noticed that you're doing your own resize, Application has a resize implementation already built in that you should consider using the `resizeTo` option. This can be the Window or an HTMLElement.

```js
const app = new Application({ resizeTo: window }); // canvas is the size of the entire window

// Optionally handle resize
app.renderer.on('resize', (width, height) => {
  // after resized, size is passed as params
});
```

_Also, I made this a discussion, which is better for support than using Issues. If you have a specific code bug, please create a new issue otherwise discussions are better. Thanks._
